readthedocs/builds/models.py

```diff
@@ -109,6 +130,16 @@ def __str__(self):
                 pk=self.pk,
             ))
 
+    @property
+    def config(self):
+        """Returns the configuration used in the last successful build."""
```

Comment:
It would be good to explain a little more what this returns and what's its type. Probably, it's good to use `:returns:` and `:rtype:` in the docstring. We do have many different types of Config objects and this will confuse us in the near future. In fact, I think this is not returning a Config object, but a simple dict :confused:

Reply:
Yep, it's just a dict